wiki/Gameplay/Kiai_time/de.md

```diff
@@ -1,24 +1,23 @@
+Während Kiai aktiv ist, leuchten alle [Hit-Objekte](/wiki/Hit_object) im Rhythmus des [Haupttakts](/wiki/Client/Beatmap_editor/Timing) auf und Sterne explodieren in der Nähe des Cursors, sobald ein [Hit-Circle](/wiki/Hit_object/Hit_circle) getroffen wurde. Anderweitig fallen sie vom Cursor herunter, wenn ein [Slider](/wiki/Hit_object/Slider) oder [Spinner](/wiki/Hit_object/Spinner) betätigt wurde.
 
-## Kiai Time beim Beatmapping
+## Beatmapping
 
-Die übliche Definition für den Kiai Time, wenn sie Beatmaps [editieren](/wiki/Client/Beatmap_editor), ist die "stärkste Stelle des Liedes". Dies ist üblicherweise der Chorus, einige Lieder besitzen jedoch keinen Chorus, haben aber jedoch immernoch ausdrucksstarke Stellen im Lied, die für Sie für die Kiai Time-Sektionen infrage kommen. Beachten Sie jedoch, dass das Spammen von Sektionen, die einen Kiai Time einleiten, zu Spams aus Fontänen führt. Solche Maps sind nicht rankbar und werden von den meisten Spielern gehasst, da sie Lags verursachen können.
+Kiai-Zeit wird häufig für den "intensivsten Teil eines Songs" verwendet, der normalerweise der Refrain ist. Da Kiai-Sektionen der ausdrucksstärkste Teil eines Songs sind, sind sie meistens etwas anspruchsvoller im Spielen als der Rest der [Beatmap](/wiki/Beatmap).
 
-## Dinge, die Sie bei der Verwendung vom Kiai berücksichtigen sollten
+## osu!taiko
 
-1. Bei **[osu!taiko](/wiki/Game_mode/osu!taiko) Maps** geben die Kiai-Sektionen 20% mehr [Punkte](/wiki/Gameplay/Score). Wenn Sie also osu!taiko spezifische [Maps](/wiki/Beatmap) erstellen, sollten Sie berücksichtigen, dass der Kiai das Gameplay mitbeeinflusst.
-2. **Konsequent sein**; es ist nervig, wenn Sie 3 Kiai-Sektionen an verschienden Sektionen innerhalb einer [Schwierigkeitsstufe](/wiki/Client/Beatmap_editor/Song_Setup) definieren und sich alle voneinander soundtechnisch unterscheiden. Des Weiteren ist es für einen [Mapper](/wiki/Beatmapping) ratsam die Kiai Time-Sektionen bei allen eigenen Schwierigkeitsstufen zu vereinheitlichen.
-3. **Benutzen Sie keine weiße Combofarben** (255,255,255); dadurch wird der Spieler geblendet und dessen Augen stark beansprucht.
-4. Es ist normal, dass eine Kiai-Sektion im Vergleich zum Rest der Map schwieriger ist. Siehe Definiton vom Kiai Time.
-5. **Die Kiai Time-Sektionen in [Gast Schwierigkeitsstufen](/wiki/Beatmap/Guest_difficulty) können sich von den Schwierigkeitsstufen des Beatmaperstellers unterscheiden**. Dies ist des öfteren der Fall, da nicht jeder denkt, dass die eine Stelle am besten für den Kiai geeignet ist; nichts desto trotz sollte sie jedoch passend zur Map definiert werden.
-6. Colourhaxing ist dazu da, um dem Kiai durch den Set aus verschiedenen Combofarben einen gewissen Touch zu geben.
+Auf [osu!taiko](/wiki/Game_mode/osu!taiko)-Beatmaps geben Kiai-Sektionen 20 % mehr [Score](/wiki/Gameplay/Score).
```

Comment:
```suggestion
Auf [osu!taiko](/wiki/Game_mode/osu!taiko)-Beatmaps geben Kiai-Sektionen 20 % mehr [Punkte](/wiki/Gameplay/Score).
```

Reply:
Differentiating between `Punkte` and `Score` is not easy. Therefore, I will use `Score` to reduce ambiguity.